crates/cadenza-eval/src/eval.rs

```diff
@@ -139,7 +220,7 @@ impl Eval for Expr {
             Expr::Literal(lit) => lit.eval(ctx),
             Expr::Ident(ident) => ident.eval(ctx),
             Expr::Apply(apply) => apply.eval(ctx),
-            Expr::Attr(attr) => attr.eval(ctx),
+            Expr::Attr(attr) => Err(Diagnostic::syntax("attribute node not supported; use @ special form").with_span(attr.span())),
```

Comment:
We need to get rid of this since it's no longer special cases - it's just another operator